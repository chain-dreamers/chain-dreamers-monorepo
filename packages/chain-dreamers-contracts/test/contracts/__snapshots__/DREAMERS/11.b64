data:application/json;base64,ZGF0YTphcHBsaWNhdGlvbi9qc29uLHsiaW1hZ2VfZGF0YSI6ICIlM2NzdmclMjB4bWxucz0naHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmcnJTIwdmlld0JveD0nMCUyMDAlMjAyNTUlMjAyNTUnJTIwd2lkdGg9JzUwMHB4JyUyMGhlaWdodD0nNTAwcHgnJTNlJTNjcGF0aCUyMGQ9J00wLDBIMjU1VjI1NUgwVjAnJTIwZmlsbD0nJTIzQ0EzNzkyJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE5NywxMDZINThDNDYsMTA2LDM2LDExNywzNiwxMzFDMzYsMTQ1LDQ2LDE1Niw1OCwxNTZIMTk3QzIwOSwxNTYsMjE5LDE0NSwyMTksMTMxQzIxOSwxMTcsMjA5LDEwNiwxOTcsMTA2JyUyMGZpbGw9JyUyMzU5NkE3NiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNzMsMjE3SDgyQzU0LDIyNCw0OCwyNTUsNDgsMjU1SDIwN0MyMDcsMjU1LDIwMSwyMjQsMTczLDIxNyclMjBmaWxsPSclMjM1OTZBNzYnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODIsMjExSDE3M1YyNTVIODJWMjExJyUyMGZpbGw9JyUyMzcyODQ5MCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J004MywyMTFWMjI1Qzk2LDIzOCwxMTEsMjQ2LDEyNywyNDZDMTQ0LDI0NiwxNTksMjM4LDE3MiwyMjVWMjExSDgzJyUyMGZpbGw9JyUyMzUwNjk3QicvJTNlJTNjcGF0aCUyMGQ9J001OCwxMzVBNjksOTksMCwxLDAsMTk3LDEzNUE2OSw5OSwwLDEsMCw1OCwxMzUnJTIwZmlsbD0nJTIzOUVBQ0I2JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTc5LDEzNUM3OSw4NiwxMDQsNDUsMTM2LDM4QTQ0LDQ0LDAsMCwwLDEyNiwzN0M4OSwzNyw1OSw4MSw1OSwxMzVDNTksMTkwLDg5LDIzMywxMjYsMjMzQTUwLDUwLDAsMCwwLDEzNiwyMzJDMTA0LDIyNSw3OSwxODUsNzksMTM1JyUyMGZpbGw9JyUyMzY0N0E4QScvJTNlJTNjcGF0aCUyMGQ9J00xOTYsMTIwQzE5NiwxMjAsMjA5LDExOCwyMTAsMTM1JyUyMGZpbGw9J25vbmUnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNTksMTIwQzU5LDEyMCw0NiwxMTgsNDUsMTM1JyUyMGZpbGw9J25vbmUnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNODMsNThDODEsNjEsODAsNjMsNzgsNjVMOTUsOTNBMjAsMjAsMCwwLDEsOTgsMTAzVjIyNUE1MSw1MSwwLDAsMCwxMDUsMjMwVjEwMkEyMCwyMCwwLDAsMCwxMDIsOTFMODMsNTgnJTIwZmlsbD0nJTIzNTk2QTc2JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE3Miw1OEwxNTMsOTFBMjEsMjEsMCwwLDAsMTUwLDEwMlYyMzBBNTAsNTAsMCwwLDAsMTU3LDIyNVYxMDNBMjEsMjEsMCwwLDEsMTYwLDkzTDE3Nyw2NVExNzUsNjIsMTcyLDU4JyUyMGZpbGw9JyUyMzU5NkE3NiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMjcsNTVDMTI3LDU1LDExMyw1NCwxMDYsNjFDMTA2LDYxLDExNSw4MiwxMjcsODJDMTQwLDgyLDE0OSw2MSwxNDksNjFDMTQyLDU0LDEyNyw1NSwxMjcsNTUnJTIwZmlsbD0nJTIzNTk2QTc2JyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE2MiwxODBDMTYyLDE4MCwxNTgsMTgwLDE1NSwxODVBNSw1LDAsMCwxLDE1MCwxODhWMTg4QTUsNSwwLDAsMSwxNDYsMTgwVjE4MEMxNDYsMTgwLDE0MCwxODAsMTM4LDE4NEMxMzYsMTg5LDEzMSwxODgsMTI3LDE4OEMxMjQsMTg4LDExOSwxODksMTE3LDE4NEMxMTUsMTgwLDEwOSwxODAsMTA5LDE4MFYxODBBNSw1LDAsMCwxLDEwNSwxODhWMTg4QTUsNSwwLDAsMSwxMDAsMTg1Qzk3LDE4MCw5MywxODAsOTMsMTgwQzkzLDE4MCw5MCwxOTYsMTI3LDE5NkMxNjUsMTk2LDE2MiwxODAsMTYyLDE4MCclMjBmaWxsPSclMjMyMjIyMjInJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTI3LDE5NkMxMTEsMTk2LDEwNywxOTIsMTA3LDE5MkMxMDcsMTkyLDExMywyMDYsMTI3LDIwNkMxNDIsMjA2LDE0OCwxOTIsMTQ4LDE5MkMxNDgsMTkyLDE0NCwxOTYsMTI3LDE5NiclMjBmaWxsPSclMjMxNzE3MTcnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTI3LDE2NkMxMTQsMTY2LDExMiwxNzUsMTExLDE4MEEzLDMsMCwwLDAsMTE0LDE4M0gxNDFBMywzLDAsMCwwLDE0NCwxODBDMTQzLDE3NSwxNDEsMTY2LDEyNywxNjYnJTIwZmlsbD0nJTIzNzI4NDkwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEyNywxNzVDMTE4LDE3NSwxMTksMTgzLDExOSwxODNIMTM2QzEzNiwxODMsMTM3LDE3NSwxMjcsMTc1JyUyMGZpbGw9JyUyMzFEMUQxRCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMjcsMTY3QzExNSwxNjcsMTEyLDE3NSwxMTIsMTc5QTIsMiwwLDAsMCwxMTUsMTgzSDExNkMxMTYsMTc4LDExOCwxNzAsMTMxLDE3MEExOSwxOSwwLDAsMSwxMzgsMTcxQzEzNiwxNjksMTMyLDE2NywxMjcsMTY3JyUyMGZpbGw9JyUyMzMzNDQ1MScvJTNlJTNjcGF0aCUyMGQ9J003OCwxMzdBMjIsMTYsMCwxLDAsMTIyLDEzN0EyMiwxNiwwLDEsMCw3OCwxMzcnJTIwZmlsbD0nJTIzNzI4NDkwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTk1LDEzN0ExMywxMywwLDAsMSwxMDQsMTI0QTEzLDEzLDAsMCwwLDEwMCwxMjNBMTMsMTMsMCwwLDAsMTAwLDE1MUExMywxMywwLDAsMCwxMDQsMTUwQTEzLDEzLDAsMCwxLDk1LDEzNyclMjBmaWxsPSclMjMwMDAwMDAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTA0LDEyNEExMywxMywwLDAsMCwxMDQsMTUwQTEzLDEzLDAsMCwwLDEwNCwxMjQnJTIwZmlsbD0nJTIzRkZGRkZGJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTEyMiwxMzRINzhBMTIsMTIsMCwwLDAsNzgsMTQwSDEyMkExMiwxMiwwLDAsMCwxMjIsMTM0JyUyMGZpbGw9JyUyM0ZGRkYwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xMzMsMTM3QTIyLDE2LDAsMSwwLDE3NywxMzdBMjIsMTYsMCwxLDAsMTMzLDEzNyclMjBmaWxsPSclMjM3Mjg0OTAnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTUwLDEzN0ExMywxMywwLDAsMSwxNTksMTI0QTEzLDEzLDAsMCwwLDE1NSwxMjNBMTMsMTMsMCwwLDAsMTU1LDE1MUExMywxMywwLDAsMCwxNTksMTUwQTEzLDEzLDAsMCwxLDE1MCwxMzcnJTIwZmlsbD0nJTIzMDAwMDAwJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTE1OSwxMjRBMTMsMTMsMCwwLDAsMTU5LDE1MEExMywxMywwLDAsMCwxNTksMTI0JyUyMGZpbGw9JyUyM0ZGRkZGRiclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNzcsMTM0SDEzM0ExMiwxMiwwLDAsMCwxMzMsMTQwSDE3N0ExMiwxMiwwLDAsMCwxNzcsMTM0JyUyMGZpbGw9JyUyM0ZGRkYwMCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J001OSwxNTZINThINTVBOSw5LDAsMCwxLDU3LDE2MUE4LDgsMCwwLDEsNDgsMTY5QTgsOCwwLDAsMSw0MCwxNjBBOCw4LDAsMCwxLDQ2LDE1Mkg0N0EyLDIsMCwwLDAsNDYsMTQ4QTEwLDEwLDAsMCwwLDQzLDE0OUExMiwxMiwwLDEsMCw2MSwxNjNDNjAsMTYxLDYwLDE1OCw1OSwxNTZWMTU2JyUyMGZpbGw9JyUyM0IxRjhGRCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00yMTIsMTQ5QTExLDExLDAsMCwwLDIwOSwxNDhBMSwxLDAsMCwwLDIwNywxNTBBMSwxLDAsMCwwLDIwOCwxNTJIMjA5QTgsOCwwLDAsMSwyMTUsMTYxQTgsOCwwLDAsMSwyMDcsMTY5QTgsOCwwLDAsMSwxOTgsMTYxQTcsNywwLDAsMSwyMDAsMTU2SDE5N0gxOTZDMTk1LDE1OSwxOTUsMTYxLDE5NCwxNjNBMTIsMTIsMCwwLDAsMjE5LDE2MkExMiwxMiwwLDAsMCwyMTIsMTQ5JyUyMGZpbGw9JyUyM0IxRjhGRCclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00zOSwxNjJBMTEsMTEsMCwwLDEsNDcsMTQ5QzMyLDE0OSwzMywxNzQsNDksMTcyQTExLDExLDAsMCwxLDM5LDE2MiclMjBmaWxsPSclMjM1RkMyQzcnLyUzZSUzY3BhdGglMjBkPSdNNTIsMTY5QzYzLDE2Niw1NiwxNTcsNTYsMTU3QzYwLDE2Niw1MiwxNjksNTIsMTY5JyUyMGZpbGw9JyUyMzVGQzJDNycvJTNlJTNjcGF0aCUyMGQ9J00yMDgsMTQ5QTExLDExLDAsMCwxLDIwNiwxNzJDMjIyLDE3NCwyMjMsMTQ5LDIwOCwxNDknJTIwZmlsbD0nJTIzNUZDMkM3Jy8lM2UlM2NwYXRoJTIwZD0nTTE5OSwxNTdDMTk5LDE1NywxOTIsMTY2LDIwMywxNjlDMjAzLDE2OSwxOTUsMTY2LDE5OSwxNTcnJTIwZmlsbD0nJTIzNUZDMkM3Jy8lM2UlM2NwYXRoJTIwZD0nTTEzNCw3MUM4OCw2OSw3NSw0Miw3MSwzMkM3Myw0MSw3Niw2NCw2NCw3MkM0OCw4Myw0MCwxMDksNDAsMTA5QzQwLDEwOSw1MSw5Niw2MSwxMDJDNjEsMTAyLDYyLDEzNCw1NywxNjBDNTcsMTYwLDc5LDEyMSw3Nyw5NkM3Nyw5Niw4OSw5NCw5NiwxMTNDOTYsMTEzLDEwMiwxMDUsMTAxLDkxQzEwMSw5MSwxMDcsOTQsMTE2LDkzQzEzNiw4NCwxMzQsNzEsMTM0LDcxJyUyMGZpbGw9JyUyMzYzNjM2MyclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xNDIsMzRDMTQyLDM0LDEyMywxMiw3MCwyOEEyOCwyOCwwLDAsMCw3MSwzMkM3NSw0Miw4OCw2OSwxMzQsNzFDMTM0LDcxLDEzNiw4NCwxMTYsOTNBNzUsNzUsMCwwLDEsMTA5LDk2QzEwOSw5NiwxMzMsMTAyLDE1NSw4NkMxNTUsODYsMTg1LDk5LDE4NSwxMzdDMTg1LDEzNywxOTgsMTExLDE5MSw5MUMyMTIsNzUsMjA5LDE0LDE0MiwzNCclMjBmaWxsPSclMjMxNTE1MTUnJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNjYsMTAyQTEzLDEzLDAsMCwwLDYyLDEwM0M2MiwxMDksNjIsMTM1LDU4LDE1N0EyMDQsMjA0LDAsMCwwLDcxLDEyN0M3MiwxMTUsNzEsMTA1LDcxLDEwM0ExMCwxMCwwLDAsMCw2NiwxMDInJTIwZmlsbD0nJTIzM0UzRTNFJy8lM2UlM2NwYXRoJTIwZD0nTTY0LDczQzUyLDgxLDQ0LDEwMCw0MiwxMDdDNDQsMTA1LDQ5LDEwMSw1NCwxMDBDNTgsOTEsNjUsNzksNzQsNzNDODAsNjksODIsNjIsODMsNTFBNTQsNTQsMCwwLDEsNzIsMzZDNzQsNDcsNzUsNjUsNjQsNzMnJTIwZmlsbD0nJTIzM0UzRTNFJy8lM2UlM2NwYXRoJTIwZD0nTTg4LDk4Vjk4Qzg4LDk4LDg4LDk5LDg4LDk5QzkxLDEwMiw5NCwxMDYsOTYsMTEyQTMwLDMwLDAsMCwwLDk5LDEwM0M5NSw5OSw5MSw5OCw4OCw5OCclMjBmaWxsPSclMjMzRTNFM0UnLyUzZSUzY3BhdGglMjBkPSdNMTM0LDc0QzEzNCw3OSwxMzEsODcsMTE2LDk0QTUzLDUzLDAsMCwxLDExMSw5NkE2Myw2MywwLDAsMCwxMjAsOTdWOTdBNjgsNjgsMCwwLDAsMTM4LDk0QzE1MCw4NywxNTAsNzgsMTUwLDc2QTkyLDkyLDAsMCwxLDEzNCw3NCclMjBmaWxsPSclMjMwMDAwMDAnLyUzZSUzY3BhdGglMjBkPSdNMTcyLDkyQzE3MCw5MiwxNjksOTMsMTY4LDk0QzE3NiwxMDEsMTg1LDExNCwxODUsMTM1QTkzLDkzLDAsMCwwLDE5MiwxMTBBNTAsNTAsMCwwLDAsMTcyLDkyJyUyMGZpbGw9JyUyMzAwMDAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xODUsMzRBNDMsNDMsMCwwLDAsMTY3LDMxQTc1LDc1LDAsMCwwLDE1MCwzM0EzMSwzMSwwLDAsMSwxNTgsMzhBODksODksMCwwLDEsMTgzLDM0SDE4NSclMjBmaWxsPSclMjMwMDAwMDAnLyUzZSUzY3BhdGglMjBkPSdNODcsMzZDODYsMzQsODYsMzMsODYsMzNWMzJWMzJDOTksMjgsMTExLDI2LDEyMiwyNkMxMjQsMjYsMTI2LDI3LDEyOSwyN0E2Nyw2NywwLDAsMCwxMDYsMjNDOTUsMjMsODMsMjUsNzEsMjlDNzEsMjksNzEsMzAsNzIsMzJDNzUsNDAsODUsNjEsMTE3LDY4Qzk3LDU5LDg5LDQzLDg3LDM2JyUyMGZpbGw9JyUyMzAwMDAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xOTQsMTA3SDYxQzYxLDExNyw1NywxNTQsODQsMTU0QzEwMywxNTQsMTA4LDEzNSwxMDgsMTM1SDE0N0MxNDcsMTM1LDE1MSwxNTQsMTcxLDE1NEMxOTgsMTU0LDE5NCwxMTcsMTk0LDEwN00xMzgsMTI2SDExN1YxMTZIMTM4VjEyNiclMjBmaWxsPSclMjMxQjFCMUInJTIwc3Ryb2tlPSclMjMwMDAnLyUzZSUzY3BhdGglMjBkPSdNNzUsMTQzQzc3LDE0Niw4MCwxNDcsODQsMTQ3Qzk3LDE0NywxMDEsMTM1LDEwMSwxMzNBNyw3LDAsMCwxLDEwOCwxMjhWMTE0SDY4QzY4LDEyMyw2OCwxMzcsNzUsMTQzJyUyMGZpbGw9JyUyM0U3RDI5QyclMjBzdHJva2U9JyUyMzAwMCcvJTNlJTNjcGF0aCUyMGQ9J00xODcsMTE0SDE0N1YxMjhBNyw3LDAsMCwxLDE1NCwxMzNDMTU0LDEzNSwxNTgsMTQ3LDE3MSwxNDdDMTc1LDE0NywxNzgsMTQ2LDE4MCwxNDNDMTg3LDEzNywxODcsMTIzLDE4NywxMTQnJTIwZmlsbD0nJTIzRTdEMjlDJyUyMHN0cm9rZT0nJTIzMDAwJy8lM2UlM2NwYXRoJTIwZD0nTTc4LDExNUg2OUM2OSwxMjYsNjksMTQ2LDg0LDE0NkExNSwxNSwwLDAsMCw5MywxNDNIOTJDNzgsMTQzLDc4LDEyNiw3OCwxMTUnJTIwZmlsbD0nJTIzRDFBRDVGJy8lM2UlM2NwYXRoJTIwZD0nTTE3NywxMTVDMTc3LDEyNiwxNzYsMTQzLDE2MiwxNDNBMTUsMTUsMCwwLDAsMTcxLDE0NkMxODYsMTQ2LDE4NiwxMjYsMTg2LDExNUgxNzcnJTIwZmlsbD0nJTIzRDFBRDVGJy8lM2UlM2NwYXRoJTIwZD0nTTE1MywxMzdIMTQ5QzE1MSwxNDEsMTU0LDE0NywxNjEsMTUwQTI5LDI5LDAsMCwxLDE1MywxMzcnJTIwZmlsbD0nJTIzMDAwMDAwJy8lM2UlM2NzdHlsZSUzZXBhdGh7c3Ryb2tlLXdpZHRoOjAuNzF9JTNjL3N0eWxlJTNlJTNjL3N2ZyUzZSIsICJuYW1lIjogIkRyZWFtZXIlMjAlMjMxMSIsICJkZXNjcmlwdGlvbiI6ICJSdW5uZXJzJTIwcnVuLCUyMGJ1dCUyMHNvbWV0aW1lcyUyMHRoZXklMjBkcmVhbS4lMjBUaGlzJTIwaXMlMjBvbmUlMjBvZiUyMHRoZWlyJTIwZHJlYW1zLiIsImF0dHJpYnV0ZXMiOiAiIn0=