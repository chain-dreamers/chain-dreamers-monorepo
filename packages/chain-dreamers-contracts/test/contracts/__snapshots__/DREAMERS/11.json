data:application/json,{"image_data": "%3csvg%20xmlns='http://www.w3.org/2000/svg'%20viewBox='0%200%20255%20255'%20width='500px'%20height='500px'%3e%3cpath%20d='M0,0H255V255H0V0'%20fill='%23CA3792'%20stroke='%23000'/%3e%3cpath%20d='M197,106H58C46,106,36,117,36,131C36,145,46,156,58,156H197C209,156,219,145,219,131C219,117,209,106,197,106'%20fill='%23596A76'%20stroke='%23000'/%3e%3cpath%20d='M173,217H82C54,224,48,255,48,255H207C207,255,201,224,173,217'%20fill='%23596A76'%20stroke='%23000'/%3e%3cpath%20d='M82,211H173V255H82V211'%20fill='%23728490'%20stroke='%23000'/%3e%3cpath%20d='M83,211V225C96,238,111,246,127,246C144,246,159,238,172,225V211H83'%20fill='%2350697B'/%3e%3cpath%20d='M58,135A69,99,0,1,0,197,135A69,99,0,1,0,58,135'%20fill='%239EACB6'%20stroke='%23000'/%3e%3cpath%20d='M79,135C79,86,104,45,136,38A44,44,0,0,0,126,37C89,37,59,81,59,135C59,190,89,233,126,233A50,50,0,0,0,136,232C104,225,79,185,79,135'%20fill='%23647A8A'/%3e%3cpath%20d='M196,120C196,120,209,118,210,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M59,120C59,120,46,118,45,135'%20fill='none'%20stroke='%23000'/%3e%3cpath%20d='M83,58C81,61,80,63,78,65L95,93A20,20,0,0,1,98,103V225A51,51,0,0,0,105,230V102A20,20,0,0,0,102,91L83,58'%20fill='%23596A76'%20stroke='%23000'/%3e%3cpath%20d='M172,58L153,91A21,21,0,0,0,150,102V230A50,50,0,0,0,157,225V103A21,21,0,0,1,160,93L177,65Q175,62,172,58'%20fill='%23596A76'%20stroke='%23000'/%3e%3cpath%20d='M127,55C127,55,113,54,106,61C106,61,115,82,127,82C140,82,149,61,149,61C142,54,127,55,127,55'%20fill='%23596A76'%20stroke='%23000'/%3e%3cpath%20d='M162,180C162,180,158,180,155,185A5,5,0,0,1,150,188V188A5,5,0,0,1,146,180V180C146,180,140,180,138,184C136,189,131,188,127,188C124,188,119,189,117,184C115,180,109,180,109,180V180A5,5,0,0,1,105,188V188A5,5,0,0,1,100,185C97,180,93,180,93,180C93,180,90,196,127,196C165,196,162,180,162,180'%20fill='%23222222'%20stroke='%23000'/%3e%3cpath%20d='M127,196C111,196,107,192,107,192C107,192,113,206,127,206C142,206,148,192,148,192C148,192,144,196,127,196'%20fill='%23171717'%20stroke='%23000'/%3e%3cpath%20d='M127,166C114,166,112,175,111,180A3,3,0,0,0,114,183H141A3,3,0,0,0,144,180C143,175,141,166,127,166'%20fill='%23728490'%20stroke='%23000'/%3e%3cpath%20d='M127,175C118,175,119,183,119,183H136C136,183,137,175,127,175'%20fill='%231D1D1D'%20stroke='%23000'/%3e%3cpath%20d='M127,167C115,167,112,175,112,179A2,2,0,0,0,115,183H116C116,178,118,170,131,170A19,19,0,0,1,138,171C136,169,132,167,127,167'%20fill='%23334451'/%3e%3cpath%20d='M78,137A22,16,0,1,0,122,137A22,16,0,1,0,78,137'%20fill='%23728490'%20stroke='%23000'/%3e%3cpath%20d='M95,137A13,13,0,0,1,104,124A13,13,0,0,0,100,123A13,13,0,0,0,100,151A13,13,0,0,0,104,150A13,13,0,0,1,95,137'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M104,124A13,13,0,0,0,104,150A13,13,0,0,0,104,124'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M122,134H78A12,12,0,0,0,78,140H122A12,12,0,0,0,122,134'%20fill='%23FFFF00'%20stroke='%23000'/%3e%3cpath%20d='M133,137A22,16,0,1,0,177,137A22,16,0,1,0,133,137'%20fill='%23728490'%20stroke='%23000'/%3e%3cpath%20d='M150,137A13,13,0,0,1,159,124A13,13,0,0,0,155,123A13,13,0,0,0,155,151A13,13,0,0,0,159,150A13,13,0,0,1,150,137'%20fill='%23000000'%20stroke='%23000'/%3e%3cpath%20d='M159,124A13,13,0,0,0,159,150A13,13,0,0,0,159,124'%20fill='%23FFFFFF'%20stroke='%23000'/%3e%3cpath%20d='M177,134H133A12,12,0,0,0,133,140H177A12,12,0,0,0,177,134'%20fill='%23FFFF00'%20stroke='%23000'/%3e%3cpath%20d='M59,156H58H55A9,9,0,0,1,57,161A8,8,0,0,1,48,169A8,8,0,0,1,40,160A8,8,0,0,1,46,152H47A2,2,0,0,0,46,148A10,10,0,0,0,43,149A12,12,0,1,0,61,163C60,161,60,158,59,156V156'%20fill='%23B1F8FD'%20stroke='%23000'/%3e%3cpath%20d='M212,149A11,11,0,0,0,209,148A1,1,0,0,0,207,150A1,1,0,0,0,208,152H209A8,8,0,0,1,215,161A8,8,0,0,1,207,169A8,8,0,0,1,198,161A7,7,0,0,1,200,156H197H196C195,159,195,161,194,163A12,12,0,0,0,219,162A12,12,0,0,0,212,149'%20fill='%23B1F8FD'%20stroke='%23000'/%3e%3cpath%20d='M39,162A11,11,0,0,1,47,149C32,149,33,174,49,172A11,11,0,0,1,39,162'%20fill='%235FC2C7'/%3e%3cpath%20d='M52,169C63,166,56,157,56,157C60,166,52,169,52,169'%20fill='%235FC2C7'/%3e%3cpath%20d='M208,149A11,11,0,0,1,206,172C222,174,223,149,208,149'%20fill='%235FC2C7'/%3e%3cpath%20d='M199,157C199,157,192,166,203,169C203,169,195,166,199,157'%20fill='%235FC2C7'/%3e%3cpath%20d='M134,71C88,69,75,42,71,32C73,41,76,64,64,72C48,83,40,109,40,109C40,109,51,96,61,102C61,102,62,134,57,160C57,160,79,121,77,96C77,96,89,94,96,113C96,113,102,105,101,91C101,91,107,94,116,93C136,84,134,71,134,71'%20fill='%23636363'%20stroke='%23000'/%3e%3cpath%20d='M142,34C142,34,123,12,70,28A28,28,0,0,0,71,32C75,42,88,69,134,71C134,71,136,84,116,93A75,75,0,0,1,109,96C109,96,133,102,155,86C155,86,185,99,185,137C185,137,198,111,191,91C212,75,209,14,142,34'%20fill='%23151515'%20stroke='%23000'/%3e%3cpath%20d='M66,102A13,13,0,0,0,62,103C62,109,62,135,58,157A204,204,0,0,0,71,127C72,115,71,105,71,103A10,10,0,0,0,66,102'%20fill='%233E3E3E'/%3e%3cpath%20d='M64,73C52,81,44,100,42,107C44,105,49,101,54,100C58,91,65,79,74,73C80,69,82,62,83,51A54,54,0,0,1,72,36C74,47,75,65,64,73'%20fill='%233E3E3E'/%3e%3cpath%20d='M88,98V98C88,98,88,99,88,99C91,102,94,106,96,112A30,30,0,0,0,99,103C95,99,91,98,88,98'%20fill='%233E3E3E'/%3e%3cpath%20d='M134,74C134,79,131,87,116,94A53,53,0,0,1,111,96A63,63,0,0,0,120,97V97A68,68,0,0,0,138,94C150,87,150,78,150,76A92,92,0,0,1,134,74'%20fill='%23000000'/%3e%3cpath%20d='M172,92C170,92,169,93,168,94C176,101,185,114,185,135A93,93,0,0,0,192,110A50,50,0,0,0,172,92'%20fill='%23000000'/%3e%3cpath%20d='M185,34A43,43,0,0,0,167,31A75,75,0,0,0,150,33A31,31,0,0,1,158,38A89,89,0,0,1,183,34H185'%20fill='%23000000'/%3e%3cpath%20d='M87,36C86,34,86,33,86,33V32V32C99,28,111,26,122,26C124,26,126,27,129,27A67,67,0,0,0,106,23C95,23,83,25,71,29C71,29,71,30,72,32C75,40,85,61,117,68C97,59,89,43,87,36'%20fill='%23000000'/%3e%3cpath%20d='M194,107H61C61,117,57,154,84,154C103,154,108,135,108,135H147C147,135,151,154,171,154C198,154,194,117,194,107M138,126H117V116H138V126'%20fill='%231B1B1B'%20stroke='%23000'/%3e%3cpath%20d='M75,143C77,146,80,147,84,147C97,147,101,135,101,133A7,7,0,0,1,108,128V114H68C68,123,68,137,75,143'%20fill='%23E7D29C'%20stroke='%23000'/%3e%3cpath%20d='M187,114H147V128A7,7,0,0,1,154,133C154,135,158,147,171,147C175,147,178,146,180,143C187,137,187,123,187,114'%20fill='%23E7D29C'%20stroke='%23000'/%3e%3cpath%20d='M78,115H69C69,126,69,146,84,146A15,15,0,0,0,93,143H92C78,143,78,126,78,115'%20fill='%23D1AD5F'/%3e%3cpath%20d='M177,115C177,126,176,143,162,143A15,15,0,0,0,171,146C186,146,186,126,186,115H177'%20fill='%23D1AD5F'/%3e%3cpath%20d='M153,137H149C151,141,154,147,161,150A29,29,0,0,1,153,137'%20fill='%23000000'/%3e%3cstyle%3epath{stroke-width:0.71}%3c/style%3e%3c/svg%3e", "name": "Dreamer%20%2311", "description": "Runners%20run,%20but%20sometimes%20they%20dream.%20This%20is%20one%20of%20their%20dreams.","attributes": ""}